static/js/util/profile_edit.js

```diff
@@ -159,6 +162,7 @@ export function boundDateField(keySet: string[], label: string, omitDay: boolean
   let onBlur = () => {
     updateValidationVisibility(keySet);
     updateProfileValidation(profile, validator);
+    sendFormFieldEvent(keySet);
```

Comment:
It might be worth adding tests to cover these calls, although this PR raises the overall coverage, the addition of these lines reduces coverage in this file by ~2%.